iamchrissmith/air_bnb_clone
# User review property

#### What does this PR do?
Users can review properties once per reservation from their dashboard.

#### Migrations
**YES**

#### Where should the reviewer start?
`spec/features/reviews/user_can_review_property_spec.rb`

#### How should this be manually tested?
1. Login as a user who has a reservation that has passed (i.e. is finished).  
2. Click Review Property.  
3. Fill in review information.  
4. Click submit.  
5. View the review on the property page.  
6. Go back to the dashboard and check that you cannot submit another review.

#### Any background context you want to provide?
I have not done the styling yet for the review form.  

#### What are the relevant tickets?
[#149453603](https://www.pivotaltracker.com/story/show/149453603)

#### Screenshots (if appropriate)

#### Questions:
  - Do Environment Variables need to be set?
  - Any other deploy steps?
